packages/dropdowns/src/Autocomplete/Autocomplete.spec.tsx

```diff
@@ -34,7 +34,7 @@ describe('Autocomplete', () => {
 
     fireEvent.click(getByTestId('autocomplete'));
 
-    expect(document.activeElement.nodeName).toBe('INPUT');
+    expect(document.activeElement!.nodeName).toBe('INPUT');
```

Comment:
Do these really add anything? The tests will fail if activeElement is null or undefined anyway?

Reply:
I see, my vim typescript setup warns about possibly null value. Neat.